src/main/java/com/danielagapov/spawn/Services/S3/S3Service.java

```diff
@@ -0,0 +1,108 @@
+package com.danielagapov.spawn.Services.S3;
+
+import com.danielagapov.spawn.DTOs.UserDTO;
+import com.danielagapov.spawn.Exceptions.ApplicationException;
+import com.danielagapov.spawn.Helpers.Logger.ILogger;
+import com.danielagapov.spawn.Mappers.UserMapper;
+import com.danielagapov.spawn.Models.User;
+import com.danielagapov.spawn.Services.User.UserService;
+import io.github.cdimascio.dotenv.Dotenv;
+import org.springframework.stereotype.Service;
+import software.amazon.awssdk.core.sync.RequestBody;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.DeleteObjectRequest;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+import java.util.UUID;
+
+
+@Service
+public class S3Service implements IS3Service {
+    private static final String BUCKET = "spawn-pfp-store";
+    private static final String CDN_BASE = Dotenv.load().get("CDN_BASE");
```

Comment:
You could add a variable here
```suggestion
    private static final String CDN_BASE = Dotenv.load().get("CDN_BASE");
    private static final String DEFAULT_PROFILE_PICTURE_URL_STRING = "something";
```